alternate-compose.yml

```diff
@@ -152,8 +154,6 @@ services:
     volumes:
       # hydroshare repository
       - ".:/hydroshare"
-      # log files
-      - "${LOGDIR}:/hydroshare/log"
       # shared location for gunicorn.sock between containers
```

Comment:
Log views will be in console, where docker-compose was executed or via docker logs